packages/x-data-grid/src/components/GridScrollArea.tsx

```diff
@@ -6,20 +6,20 @@ import {
   unstable_useEventCallback as useEventCallback,
 } from '@mui/utils';
 import { styled } from '@mui/system';
-import { getTotalHeaderHeight, fastMemo, useTimeout } from '@mui/x-data-grid/internals';
-import {
-  GridEventListener,
-  GridScrollParams,
-  getDataGridUtilityClass,
-  gridClasses,
-  gridDensityFactorSelector,
-  useGridApiContext,
-  useGridApiEventHandler,
-  useGridSelector,
-  gridColumnsTotalWidthSelector,
-} from '@mui/x-data-grid';
-import { DataGridProProcessedProps } from '../models/dataGridProProps';
+import { DataGridProcessedProps } from '../models/props/DataGridProps';
 import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+import { getDataGridUtilityClass, gridClasses } from '../constants';
+import { useGridApiContext } from '../hooks/utils/useGridApiContext';
+import { useGridApiEventHandler } from '../hooks/utils/useGridApiEventHandler';
+import { useGridSelector } from '../hooks/utils/useGridSelector';
+import { gridDimensionsSelector } from '../hooks/features/dimensions/gridDimensionsSelectors';
+import { gridDensityFactorSelector } from '../hooks/features/density/densitySelector';
+import { gridColumnsTotalWidthSelector } from '../hooks/features/columns/gridColumnsSelector';
+import { GridScrollParams } from '../models/params/gridScrollParams';
+import { GridEventListener } from '../models/events';
+import { useTimeout } from '../hooks/utils/useTimeout';
```

Comment:
We have it in `@mui/utils/useTimeout` now I think.

Reply:
Done, also for the `useLazyRef` and `useOnMount` hooks.
Plus, did the same in `@mui/x-tree-view`